pom.xml

```diff
@@ -2732,8 +2732,7 @@
             </activation>
             <properties>
                 <maven.compiler.release>8</maven.compiler.release>
-                <!-- illegal-access=permit is for cglib (jDBI) -->
```

Comment:
We still need to support jdk 8 and 11. I suspect you need a new profile for jdk 16.